src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: 1024px;
```

Comment:
you shouldn't hardcode max-width
it should be calculated according to the width and number of the product cards